packages/domainproperties/src/components/DomainRowExpandedOptions.tsx

```diff
@@ -85,10 +86,16 @@ export class DomainRowExpandedOptions extends React.Component<IDomainRowExpanded
             <div className='domain-row-container'>
                 <div className='domain-row-handle'/>
                 <div className='domain-row-container-expanded'>
-                    {this.typeDependentOptions()}
-                    <NameAndLinkingOptions index={index} field={field} onChange={onChange}/>
+                    <Col xs={12}>
+                        {this.typeDependentOptions()}
+                    </Col>
+                    <Col xs={12} lg={10}>
```

Comment:
why the lg=10 here? just curious.

Reply:
I just thought these options looked odd stretching across the larger screens.  Unlike the fields in the sections above and below them.

Reply:
Sounds good.